StarWars.Android/Resources/layout/characters_fragment.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout 
+        xmlns:android="http://schemas.android.com/apk/res/android" 
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:local="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:paddingHorizontal="16dp"
+        android:paddingVertical="64dp"
+        android:background="#000000" >
+<!--    <TextView
+          android:layout_width="match_parent"
+          android:layout_height="match_parent"
+          android:textSize="20sp"
+          android:textColor="#FFE81F"
+          android:background="#111111"
+          android:fontFamily="@font/starjedi"
+          local:MvxBind="StarWars" />-->
+    <MvvmCross.DroidX.RecyclerView.MvxRecyclerView
+            android:id="@+id/charactersList"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            local:MvxBind="ItemsSource Characters; ItemClick SelectCharacterCommand"
```

Comment:
Next: Let's get rid of xml binding. It's hard to support.
I'd suggest you to use bindings in code.